charts/postgres-operator/crds/operatorconfigurations.yaml

```diff
@@ -91,6 +91,9 @@ spec:
               etcd_host:
                 type: string
                 default: ""
+              etcd_host_version:
+                type: string
+                default: "v2"
```

Comment:
@Lucasgranet please pay attention that's able to set any Patroni environment variable via `Env` section of the `Postgresql` spec now.

Reply:
https://postgres-operator.readthedocs.io/en/latest/administrator/#via-postgres-cluster-manifest

Reply:
https://postgres-operator.readthedocs.io/en/latest/administrator/#via-postgres-cluster-manifest

Reply:
Hello @dmvolod,

We need this change because it's the only way to disable the use of Kubernetes as DCS for Patroni.
Current code force the usage of etcdv2 (which is now deprecated) that's why I suggest this change to support etcdv3.

See: https://github.com/zalando/postgres-operator/pull/1919/files#diff-e9c5b05262207b16b959659256e773d9158a1710b89d1b73eca6453888563130R867

Reply:
Thanks for explanation. Have you mind to add enum validation for possible options on the CRD level?

Reply:
Added

Reply:
Since we seem to be able to set username and password from Postgres CR, I agree, this seems to be the better solution.
Thanks @Lucasgranet !